administrator/language/de-DE/plg_system_stats.ini

```
@@ -5,24 +5,24 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_SYSTEM_STATS="System - Joomla!-Statistikerhebung"
-PLG_SYSTEM_STATS_BTN_NEVER_SEND="Niemals"
-PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Immer"
+PLG_SYSTEM_STATS_BTN_NEVER_SEND="Nein"
+PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Ja, Joomla! helfen"
 PLG_SYSTEM_STATS_BTN_SEND_NOW="Einmalig"
 PLG_SYSTEM_STATS_INTERVAL_LABEL="Intervall (Stunden)"
 PLG_SYSTEM_STATS_LABEL_CMS_VERSION="CMS-Version"
 PLG_SYSTEM_STATS_LABEL_DB_TYPE="DB-Typ"
 PLG_SYSTEM_STATS_LABEL_DB_VERSION="DB-Version"
-PLG_SYSTEM_STATS_LABEL_MESSAGE_TITLE="Joomla! bittet um Erlaubnis einige Basisstatistiken zu erheben"
+PLG_SYSTEM_STATS_LABEL_MESSAGE_TITLE="Helfen, Joomla! besser zu machen."
 PLG_SYSTEM_STATS_LABEL_PHP_VERSION="PHP-Version"
 PLG_SYSTEM_STATS_LABEL_SERVER_OS="Server OS"
 PLG_SYSTEM_STATS_LABEL_UNIQUE_ID="Eindeutige ID"
 PLG_SYSTEM_STATS_MODE_LABEL="Modus"
 PLG_SYSTEM_STATS_MODE_OPTION_ALWAYS_SEND="Immer senden"
 PLG_SYSTEM_STATS_MODE_OPTION_NEVER_SEND="Niemals senden"
-PLG_SYSTEM_STATS_MODE_OPTION_ON_DEMAND="Auf Aufforderung"
+PLG_SYSTEM_STATS_MODE_OPTION_ON_DEMAND="Nochmal fragen"
 PLG_SYSTEM_STATS_MSG_ALLOW_SENDING_DATA="Joomla!-Statistikerhebung aktivieren?"
-PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="Das Joomla!-Projekt würde gerne die Installationsbasis und die Endbenutzerumgebungen besser kennen, damit gezielt hierfür entwickelt werden kann. Dafür wurde dieses Plugin entwickelt, um die minimal benötigten Daten an einen zentralen vom Joomla!-Projekt kontrollierten Server senden zu können. Dabei werden zu <strong>keiner Zeit</strong> persönlichen Daten ermittelt. Die getroffenen Einstellungen können später unter „Erweiterungen“ &rarr; „Plugins“ &rarr; „System - Joomla!-Statistikerhebung“ geändert werden."
-PLG_SYSTEM_STATS_MSG_WHAT_DATA_WILL_BE_SENT="Hier klicken um die zu sendenden Daten anzusehen."
+PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="Das Joomla!-Projekt würde gerne die Kompatibilität steigern und benötigt dafür Informationen zur Installationsbasis und den Endbenutzerumgebungen. Alle Daten werden anonymisiert und an einen vom Joomla!-Projekt zentral gesteuerten Server übermittelt. Die Einstellungen können später unter „Verwalten“ &rarr; „Plugins“ &rarr; „System - Joomla!-Statistikerhebung“ geändert werden."
```

```suggestion
PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="Das Joomla!-Projekt würde gerne Systemstatistiken erheben um daran zukünftige Entwicklungen auszurichten. Alle Daten werden anonymisiert und an einen vom Joomla!-Projekt zentral gesteuerten Server übermittelt. Die Einstellungen können später unter „Verwalten“ &rarr; „Plugins“ &rarr; „System - Joomla!-Statistikerhebung“ geändert werden."
```

Das ist vielleicht etwas klarer was gemacht wird.